index.js

```diff
@@ -1 +1,100 @@
 // implement your API here
+const express = require('express');
+
+const db = require('./data/db');
+
+const server = express();
+
+server.use(express.json());
+
```

Comment:
I think you forgot to use cors after downloading it

Reply:
I took it out and forgot to add it back in because I was getting an error in console. I installed it, followed the instructions in the README to add it to the project, and it was causing the server to crash. I'll double check what the error was.